packages/apps/src/Menu/index.tsx

```diff
@@ -28,16 +28,16 @@ interface Props {
 function createExternals (t: TFunction): ItemRoute[] {
   return [
     {
-      href: 'https://github.com/polkadot-js/apps',
+      href: 'https://github.com/aleph-zero-foundation',
       icon: 'code-branch',
       name: 'github',
       text: t<string>('nav.github', 'GitHub', { ns: 'apps-routing' })
     },
     {
-      href: 'https://wiki.polkadot.network',
+      href: 'https://alephzero.org/',
       icon: 'book',
       name: 'wiki',
```

Comment:
If we change this to `homepage`, this should break sth right? As those are likely references in some CSS files?

Reply:
Well, this shouldn't break anything. From what I understand we want to change it back in the future to linking to wiki. Actually I could change this and `text` to `t<string>('nav.homepage', 'Homepage', { ns: 'apps-routing' })` and everything should be working and showing "homepage" in all languages.

Reply:
So I went with the least invasive option to change it back in the future. Is it fine with you?

Reply:
Yes sure,